test/data/testTD.py

```diff
@@ -33,6 +33,18 @@ class GenericTDTest(unittest.TestCase):
             "Singlet-Ag",
         ]
     sumofsec = 1.0
+    
+    @skipForParser('ADF', 'excited_states_method not yet implemented')
+    @skipForParser('DALTON', 'excited_states_method not yet implemented')
+    @skipForParser('FChk', 'etrotats are not yet implemented')
+    @skipForParser('GAMESS', 'excited_states_method not yet implemented')
+    @skipForParser('GAMESSUK', 'excited_states_method not yet implemented')
+    @skipForParser('Jaguar', 'excited_states_method not yet implemented')
+    @skipForParser('NWChem', 'excited_states_method not yet implemented')
+    @skipForParser('QChem', 'etrotats are not yet implemented')
+    def testmetadata(self):
+        """Did we parse an excited states method?"""
+        assert "excited_states_method" in self.data.metadata
```

Comment:
Could you make this more specific?

(Some of our `dvb_td` outputs might actually be TDA and not full TD-DFT.)

There are also a few `water_cis` outputs, but those might be GAMESS only.

Reply:
Yeah sure, I'll work on it